cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/declarations/EnumConstantDeclaration.kt

```diff
@@ -33,6 +34,5 @@ import de.fraunhofer.aisec.cpg.graph.statements.expressions.Expression
  * explicit initializer value.
  */
 class EnumConstantDeclaration : ValueDeclaration(), HasInitializer {
-
-    override var initializer: Expression? = null
+    @AST override var initializer: Expression? = null
```

Comment:
Isn't this already part of the value declaration's initialiser? Do we derive AST annotations from parent nodes?

Reply:
No, value declarations have no initializer, but `HasInitializer`does but has no `@AST` Annotation, should we move it there?

Reply:
Ah ok. The annotation will probably not work on interfaces